build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/tasks/BuildCommitDistribution.kt

```diff
@@ -80,6 +80,7 @@ abstract class BuildCommitDistribution @Inject internal constructor(
         }
         execOps.exec {
             commandLine(*getBuildCommands())
+            environment(getBuildEnvs())
```

Comment:
What did we pass here before? `System.getenv())`? Are we losing any environment variables now?